### PR TITLE
create 404.html page

### PR DESCRIPTION
**Pull request by Ruchika4**
Create 404.html page. Milo provides a 404 page template. Adding it to CC repo.
Resolves: MWPW-128624

I have kept script path as cclibs/scripts/scripts.js as file is moved to this path in PR https://github.com/adobecom/cc/pull/11

TestURLs:
N/A

### CODE REVIEW

**Comment by Ruchika4** [comment]
This PR is dependent on PR https://github.com/adobecom/cc/pull/11 Based on the path of scripts.js updated in that PR, this PR will need to be updated. Hence this PR cannot be merged till the above mentioned PR is merged.
Adding do not merge label till then.

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>404</title>
+  <meta name="template" content="404" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <script src="cclibs/scripts/scripts.js" type="module"></script>
```
Comment (aishwaryamathuria) [inline 404.html:8]: cclibs is not used in the PR anymore. We are using cc

Comment (aishwaryamathuria) [inline 404.html:8]: cclibs is not used in the PR anymore. We are using cc. The script path would need to be changed accordingly.

**Review by aishwaryamathuria — CHANGES_REQUESTED** [review]

File: 404.html (hunk 1 above)
Comment (Ruchika4) [reply 404.html]: @aishwaryamathuria thanks for confirming. I have updated the path.

**Review by auniverseaway — APPROVED** [review]

**Review by aishwaryamathuria — APPROVED** [review]

**Pull request merged**